### PR TITLE
chore: add ./script/rebase-stage0.sh

### PR DESCRIPTION
**Pull request by nomeata**
heavily based on an script by Kim.

### CODE REVIEW

**Comment by nomeata** [comment]
It worked here in one simple test, more testing will happen in the wild, I fear :)

File: script/aux/rebase-editor.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -x
```
Comment (nomeata) [inline script/aux/rebase-editor.sh:3]:
```suggestion
```

File: script/rebase-stage0.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 [-nix] <branch-to-rebase-onto>"
+    exit 1
+fi
+
+BRANCH_TO_REBASE="$1"
+REPO_ROOT=$(git rev-parse --show-toplevel)
+# Run git rebase in interactive mode, but automatically edit the todo list
+# using the defined GIT_SEQUENCE_EDITOR command
+GIT_SEQUENCE_EDITOR="$REPO_ROOT/script/aux/rebase-editor.sh" git rebase -i "$BRANCH_TO_REBASE"
```
Comment (Kha) [inline script/rebase-stage0.sh:23]: Could just forward all args (`$@`?) to allow for further flags or rebase target

**Comment by Kha** [comment]
Let's also move update-stage0 into aux/

**Review by Kha — APPROVED** [review]

**Comment by nomeata** [comment]
Hehe, guess I cannot call a directory `aux` on Windows. Will rename to `lib`.

**Pull request merged**